docs/user_guides/templates/scene_queries.mdx

```diff
@@ -21,11 +21,11 @@ query_pipeline.update(&island_manager, &rigid_body_set, &collider_set);
 // Scene queries can now be executed accurately.
 ```
 
-:::note
-Right now, the query pipeline takes the `IslandManager` and `RigidBodySet` as arguments. These two arguments
-may look superfluous, especially if the colliders are not attached to rigid-bodies. Another update method that only
-takes a `ColliderSet` as its only argument will be added in the future.
-:::
```

Comment:
I removed previous note entirely for several reasons:
- `IslandManager` is no longer a parameter to `update()`
- In my opinion it's not the place to mention future updates, especially "minor" ones
- I'd like to contribute to add the teased function (update only colliders).